devices/ubisys/c4_5504.json

```diff
@@ -3,8 +3,8 @@
   "uuid": "c4d31091-6713-4c79-af25-5e9830d1e9a0",
   "manufacturername": "ubisys",
   "modelid": "C4 (5504)",
-  "vendor": "ubisys",
-  "product": "C4 (5504) control unit",
+  "vendor": "ubisys ",

```

Comment:
One space too much? :)

Reply:
Probably an issue when I used the search and replace to remove the uppercase letter